### PR TITLE
fixes issue #98

### PR DESCRIPTION
**Pull request by Ashurajput02**
### Pull Request Title
feat(clnapp): fixes issue #98

### Description
Added a splash screen in the app using [flutter_native_splash](https://github.com/jonbhanson/flutter_native_splash). The splash screen now consists of the app logo on a black background.

### Checklist
- [x] I have tested my changes
- [x] I have updated the documentation
- [x] I have added necessary comments

### Screenshots (if applicable)

https://github.com/dart-lightning/lndart.clnapp/assets/119595975/fc754676-b0f1-47cd-b7eb-f31e130755f7

---

### Commits
- `feat(clnapp): fixes issue #98`
  - Added a splash screen in the app using [flutter_native_splash](https://github.com/jonbhanson/flutter_native_splash).
  - The splash screen now consists of the app logo on a black background.

### CODE REVIEW

**Comment by Ashurajput02** [comment]
@Harshit933 
greetings sir.
Sir , please go through it

**Review by vincenzopalazzo — COMMENTED** [review]
we can add the icon of the app as spleshscreen?

**Comment by Ashurajput02** [comment]
> we can add the icon of the app as spleshscreen?

Yes I added the main logo of the repo as splash screen on black background. 
Do you want to add this ⚡ icon instead of this?

**Comment by Ashurajput02** [comment]
> > we can add the icon of the app as spleshscreen?
> 
> Yes I added the main logo of the repo as splash screen on black background. Do you want to add this ⚡ icon instead of this?

See the video around 0:12

**Comment by Ashurajput02** [comment]
@vincenzopalazzo for those deprecations I guess either we need to wait for the native flutter speech to give updates or we can even built the splash screen from the scratch itself sir m all we need is to change the home to that splash screen and push. Navigator. Of(context). Pushreplacement() to homepage in splashscreen.Dart

**Comment by Ashurajput02** [comment]
@vincenzopalazzo 
seems so that the error is basically beacuse I didnt integrated with core-lightning

**Comment by Ashurajput02** [comment]
it is now showing all checks passed .
furthermore ,I am setting the project on Linux as well . Let me know if it works , else i will raise the PR again once it runs on my linux
@Harshit933

**Comment by Harshit933** [comment]
Tested on my mobile. Works good. But one question I have is why can't we just build a splashscreen from scratch? It would reduce the number of dependencies.

**Comment by Ashurajput02** [comment]
> Tested on my mobile. Works good. But one question I have is why can't we just build a splashscreen from scratch? It would reduce the number of dependencies.

In fact building a splash screen from the scratch will look more beautiful and lovely than this. Here in this we can't even change the layout. If we build from the scratch we will have full freedom to add the image, tag line and animations as well. 
I already have made one. Let me modify a bit and we can merge that one rather than of this. 
Between can you please provide me some more clear picture of the logo .

**Comment by Harshit933** [comment]
> I already have made one. Let me modify a bit and we can merge that one rather than of this. 

That would be nice but please wait for @vincenzopalazzo's review.

> Between can you please provide me some more clear picture of the logo .

I don't have it.

**Comment by Ashurajput02** [comment]
> > I already have made one. Let me modify a bit and we can merge that one rather than of this.
> 
> That would be nice but please wait for @vincenzopalazzo's review.
> 
> > Between can you please provide me some more clear picture of the logo .
> 
> I don't have it.

Sure 👍

File: flutter_native_splash.yaml [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+    # parameters from above.
+    #image_dark: assets/android12splash-invert.png
+    #color_dark: "#042a49"
+    #icon_background_color_dark: "#eeeeee"
+
+  # The android, ios and web parameters can be used to disable generating a splash screen on a given
+  # platform.
+  #android: false
+  #ios: false
+  #web: false
```
Comment (vincenzopalazzo) [inline flutter_native_splash.yaml:79]: idk but looks like it is working just for android? we should support other devices too

**Review by vincenzopalazzo — CHANGES_REQUESTED** [review]
My review is not addressed, you find the icon of the application inside the icon directory of the applications

e.g: the android device has a icon directory

File: flutter_native_splash.yaml (hunk 1 above)
Comment (Ashurajput02) [reply flutter_native_splash.yaml:79]: I guess there is some issue with some package installation . I will look into the matter shortly after my midsems. have to look this on my Fiend's MAC.

**Comment by vincenzopalazzo** [comment]
Closing because look inactive

**Pull request closed without merging**